packages/elements/src/Header/getDefaultHeaderHeight.tsx

```diff
@@ -5,15 +5,16 @@ import type { Layout } from '../types';
 export default function getDefaultHeaderHeight(
   layout: Layout,
   modal: boolean,
-  statusBarHeight: number
+  statusBarHeight: number,
+  fullScreenModal = false,
```

Comment:
I think this arg is unnecessary. Just pass `!fullScreenModal` for the `modal` argument. Can rename the argument to `modalPresentation` or something similar to make it more obvious.